scheduler/src/cook/kubernetes/api.clj

```diff
@@ -82,16 +82,20 @@
   (when-not (synthetic-pod? pod-name)
     pod-name))
 
-(defn assoc-uuids
-  "Returns event-map with added fields :job-uuid (based on pod-name) and :instance-uuid (if pod is not synthetic)"
+(defn assoc-fields
```

Comment:
I think we should rename to `pod-name->job-map` have it return a map and then call merge on the map where we use it

Reply:
Works for me

Reply:
SGTM